lib/puppet/pops/loader/ruby_function_instantiator.rb

```diff
@@ -14,17 +14,17 @@ class Puppet::Pops::Loader::RubyFunctionInstantiator
   #
   def self.create(loader, typed_name, source_ref, ruby_code_string)
     unless ruby_code_string.is_a?(String) && ruby_code_string =~ /Puppet\:\:Functions\.create_function/
-      raise ArgumentError, "The code loaded from #{source_ref} does not seem to be a Puppet 4x API function - no create_function call."
+      raise ArgumentError, _("The code loaded from %{source_ref} does not seem to be a Puppet 4x API function - no create_function call.") % { source_ref: source_ref }
```

Comment:
@Magisus create_function looks like a function name and likely should be left in english